### PR TITLE
docs: add a link to documentation of AwkwardArray in Julia

### PR DESCRIPTION
**Pull request by ianna**
AwkwardArray in Julia documentation is created by Documenter.jl.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -93,6 +93,15 @@ Spotted a typo in the documentation? Want to add to the codebase? The contributi
 :class-card: admonition warning
 
 {fas}`history` Need the documentation for version 1 of Awkward Array? Click this card.
+
+:::
+
+:::{grid-item-card}
```
Comment (ianna) [inline docs/index.md:99]: The card looks fine - as expected.

**Review by ianna — COMMENTED** [review]
@jpivarski - I'm not sure why the packaging test on ubuntu fails.

**Comment by agoose77** [comment]
@ianna looks like we're pulling in a NumPy beta, which is not yet supported!

**Comment by jpivarski** [comment]
For now, we can ignore that test failure (or prevent NumPy 2.0.0b1 from being installed, only to have to reverse it later?), particularly for this PR, which is a documentation PR.

@tcawlfield is looking into #2936 right now. Hopefully, the changes needed are small and can be confined to nplike. The first issue, `numpy.AxisError` moving to some other path, would be easy to fix. Before making major changes, like putting an upper cap on NumPy version, we need to get a sense of how many issues there are and how long it will take.

**Comment by jpivarski** [comment]
Note to self: I want to find a three-dots inline Julia icon, as a replacement for the

```
{fas}`history`
```

If neither of us can find such a thing, we can call this PR done.

**Comment by ianna** [comment]
I only found an ellipsis icon among `{fas}`

**Comment by jpivarski** [comment]
Ha! It worked!

image

I'd say it's ready to merge now. The tests won't all run, since this PR doesn't touch code.

**Pull request merged**